zhaquirks/sengled/e1e-g7f.py

```diff
@@ -0,0 +1,195 @@
+"""Sengled E1E-G7F device."""
+
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+import zigpy.types as t
+from zigpy.zcl.clusters.general import (
+    Basic,
+    Groups,
+    Identify,
+    LevelControl,
+    OnOff,
+    PollControl,
+    PowerConfiguration,
+)
+
+from .. import Bus
+from ..const import (
+    ARGS,
+    COMMAND,
+    COMMAND_OFF,
+    COMMAND_ON,
+    COMMAND_STEP,
+    DEVICE_TYPE,
+    DIM_DOWN,
+    DIM_UP,
+    DOUBLE_PRESS,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    LONG_PRESS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+    SHORT_PRESS,
+    TURN_OFF,
+    TURN_ON,
+    ZHA_SEND_EVENT,
+)
+
+
+class SengledE1EG7FOnOffCluster(CustomCluster, OnOff):
+    """Sengled E1E-G7F OnOff cluster."""
+
+    cluster_id = OnOff.cluster_id
```

Comment:
@Adminiuga Is this line necessary? I'm not super good with Python, but my understanding is attributes are not inherited, but it seemed like other examples I saw of extending existing clusters did not have this.

Reply:
Nah, that particular one is not needed. It is inherited from the OnOff class. I know there're quirks which define it, but strictly speaking is not needed.

Reply:
Okay, thanks. I'm going to remove this and the LevelControl one since they are inherited. I don't like leaving code that serves no purpose.

Reply:
Done